packages/scripts/README.md

```diff
@@ -0,0 +1,106 @@
+
+```shell
+yarn changelogupdater.js excerpt
+```
+
+### Back
+
+It creates the blank placeholder for the next release. Provided the (next) version, it will be added to the placeholder.
+
+```shell
+yarn changelogupdater.js back ${version}
+```
+
+## addon command
+
+This script creates and configure a project using the Volto generator and configures it to work with the current add-on.
+Useful when testing an add-on in a CI environment in isolation, or creating a vainilla project with the add-on already configured and ready to use, eg. for demo purposes or for simple add-on development.
+
+### clone (git)
+
+It clones the add-on into a vanilla project generated by the script.
+
+`npx -p @plone/scripts addon clone [options] <source> [destination]`
+
+    Usage: addon clone [options] <source> [destination]
+
+    clone a repository into a newly created directory
+
+    Options:
+      -p, --private          set if the repo is private, then GITHUB_TOKEN is used
+      -b, --branch <branch>  set the repo branch, defaults to main
+      -c, --canary           downloads latest Volto canary (alpha) version
+      -h, --help             display help for command
+
+This next command, would download the `volto-blocks-grid` add-on from its git repo, branch `main` and will generate a project with the latest Volto canary (alpha) version.
+
+```shell
+npx -p @plone/scripts addon clone https://github.com/kitconcept/volto-blocks-grid.git --branch main --canary
+```
+
+This will create a directory named `addon-testing-project` (this is a sensible default, but you can specify a custom one) and will bootstrap a new project using Volto's standard project generator.
+It will adjust the configuration of this project to setup the add-on in the project using `mrs-developer` and the git URL given to fetch the add-on contents.
+You can specify the branch to be used, if the project should use the latest alpha available.
+There is an option for private repos as well, in that case, it will use the `GITHUB_TOKEN` present in your environment variables to fetch it.
+
+After this, as a developer you can use the usual project commands to run tests (unit, linting, acceptance) inside the generated addon-testing-project`.
+You can configure the CI of your choice for automated testing, you can take a look at how it's done in: https://github.com/kitconcept/volto-blocks-grid/tree/main/.github/workflows
+
+The idea is to issue commands inside the generated `addon-testing-project` project and do your checks.
+Take special care on how to pass down to the `npx` command the current PR branch.
+
+### clone local
+
+You can also clone the local add-on using:
+
+```shell
+npx -p @plone/scripts addon clone .
+```
+
+This only works if you execute the command from the root of your add-on directory.
+
+### consolidate
+
+It consolidates the changes made in the previously generated project with the `clone` command into the local add-on.
```

Comment:
I'm not sure I understand what that means. Can you say more about what it does / when I would want to use it?

Reply:
I improved the explanation all over the place. Sorry it was awful...